index.js

```diff
@@ -6,9 +6,7 @@ module.exports = function () {
     return function () {};
   }
   var active = document.activeElement;
-  var ranges = Array.apply(Array, {
-    length: selection.rangeCount
-  }).map(function(range, index) {
+  var ranges = Array.apply(Array, [selection.rangeCount]).map(function(range, index) {
```

Comment:
@jory In the first example console.log is not invoked. This is the problem with `Array(5).map`. That just des not work as it should.